spinta/datasets/commands/check.py

```diff
@@ -5,6 +5,17 @@
 
 @commands.check.register()
 def check(context: Context, dataset: Dataset):
+    rc = context.get('rc')
+    filename = rc.get('filename')
+    if filename:
+        if len(context.get('store').manifest.datasets) > 1:
+            raise Exception(f'File contains more than one dataset.')
```

Comment:
Use `MultipleDatasetsError` exception from `spinta/exceptions.py` (create a new class for this).